docs/federate.md

```diff
@@ -0,0 +1,141 @@
+## When to use local query federation
+There are two main reasons to deploy local query federation:
+
+- **Case 1**: one-way federation. You have (at least) one [local neurobagel
+node](infrastructure.md) and you want your users to be able to search
+the data in the local node alongside all the publicly
+visible data in the neurobagel network.
+- **Case 2**: internal federation. You have two or more local neurobagel
+nodes (e.g. for data from different groups in your institute)
+and you want your local users to search across all of them.
+
+![Local federation scenarios](imgs/local_federation_architecture.jpg)
+
+Note that these cases are not mutually exclusive. 
+Any local neurobagel nodes you deploy will only be visible to users
+inside of your local network (internal federation).
+
+## When not to use local query federation
+Query federation is not necessary, if you:
+
+- **only want to query public neurobagel nodes**:
+  Existing public nodes in the neurobagel network are accessible
+  to everyone via our public query tool (e.g. on [query.neurobagel.org](https://query.neurobagel.org/))
+  and you can choose to also make them available to your local deployment
+  through one-way federation.
```

Comment:
🍒 :
```suggestion
  to everyone via our public query tool (e.g. on [query.neurobagel.org](https://query.neurobagel.org/)),
  meaning you can run federated queries over these graph databases without any additional local setup.
```
To avoid confusion, I wouldn't mention one-way federation here as it's not relevant to the case of "When **not** to use local query federation"